### ISSUE TITLE
Drop Author and Committer exercise

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/git-novice#514]
The "Author and Committer" exercise in http://swcarpentry.github.io/git-novice/04-changes/index.html is not suitable for the novice lesson. We should move it to extras.

**Comment by iglpdc** [comment]
Agreed! We could also remove it at all.

As an alternative exercise, what about commiting an empty dir? This is useful when one wants to layout a project (for example, have a `data` folder, that holds the data without the data being committed to the repo). It is also quite common around the internet and we use it in our lessons.

Apart from the practical benefit, it could help to understand Git better by explaining briefly why you cannot commit/track an completly empty folder.

**Comment by alexanderzimmerman** [comment]
@iglpdc , why is an alternative exercise needed? A good first step might be just to remove this exercise.

**Comment by iglpdc** [comment]
@alexanderzimmerman, I agree. We could/should split this in two issues: drop the exercise or move it to the extras (this issue), and another one to be opened to add a new exercise with the empty dir.
